java-client/src/main/java/org/apache/pegasus/apps/batch_get_request.java

```diff
@@ -1,469 +0,0 @@
-/*
```

Comment:
These auto-generated file should add into `.gitignore`, other same

Reply:
+1

Reply:
done